src/calculator/routes.ts

```diff
@@ -29,6 +29,15 @@ calculatorRoutes.get('/twitter/user/:id', validate('twitterUserCredibility'), fu
     })
 })
 
+calculatorRoutes.get('/user/scrape', validate('scrapperTwitterUserCredibility'), function(req, res) {
+  const errors = validationResult(req)
+  if (!errors.isEmpty()) {
```

Comment:
@german1608 Sorry to bother you man, should I also include this line in my tasks?

Reply:
Hey. Yeah try to do so. The error schema is described on the attached confluence page. look below how validations are made